blueprints/ranim2.py

```diff
@@ -0,0 +1,72 @@
+from flask import Blueprint, jsonify,request
+import requests
+
+ranim2= Blueprint("ranim2", __name__, url_prefix="/api/ranim")
+
+@ranim2.route("/user/<user_id>/todos", methods=["GET"])
+def todos_flask(user_id):
+    response= requests.get(f"https://jsonplaceholder.typicode.com/users/{user_id}/todos")
+    
+    if request.method == "GET":
+        values= request.args
+    else:
+        return jsonify(error="Invalid method!"), 405
+    
+    if user_id is None:
+        return jsonify(error="Missing `user id`"), 400
+    
+    if int(response.status_code)//100 !=2:
+         return jsonify(error="Invalid request!")
+    
+    return response.json()
+
+@ranim2.route("/user/<user_id>/posts", methods=["GET"])
+def posts_flask(user_id):
+    response= requests.get(f"https://jsonplaceholder.typicode.com/users/{user_id}/posts")
+    
+    if request.method == "GET":
+        values= request.args
+        
+    else:
+        return jsonify(error="Invalid method!"), 405
+    
+
+    if user_id is None:
+        return jsonify(error="Missing `user id`"), 400
+    
+    if int(response.status_code)//100 !=2:
+         return jsonify(error="Invalid request!")
+    
+    return response.json()
+
+@ranim2.route("/user/<user_id>/posts/comments", methods=["GET"])
+def comments_flask(user_id):
+    response= requests.get(f"https://jsonplaceholder.typicode.com/posts/{user_id}/comments")
```

Comment:
This is incorrect. The id here is a post_id and not a user_id
From the guide:
![image](https://github.com/raldebsi/task-flask-common/assets/121934265/525b69bd-b831-4251-8247-fe99bc2f253f)